code/modules/client/preference_setup/background/01_species.dm

```diff
@@ -49,6 +49,8 @@
 	. += "<tr><td colspan=3><center>"
 	for(var/s in get_playable_species())
 		var/decl/species/list_species = get_species_by_key(s)
+		if(!istype(list_species))
+			continue // Runtime observed in a round where the playable species list was edited.
```

Comment:
it was edited to the wrong value because i was silly, this check is pointless and i fixed it in-game immediately